client/src/components/layout/CommentForm.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react"
+import getCurrentUser from "../../services/getCurrentUser.js"
+
+import translateServerErrors from "../../services/translateServerErrors.js"
+import ErrorList from "./ErrorList.js"
+
+const Comment = (props) => {
+  const [comment, setComment] = useState({
+    title: "",
+    content: "",
+    userId: "",
+    tripId: "",
+  })
+  const [errors, setErrors] = useState([])
+
+  const [currentUser, setCurrentUser] = useState({
+    id: "",
+    email: "",
+    userName: "",
+  })
+  useEffect(() => {
+    getCurrentUser()
+      .then((user) => {
+        setCurrentUser(user)
+      })
+      .catch(() => {
+        setCurrentUser(null)
+      })
+  }, [])
```

Comment:
hmmmm we should be able to get the currentUser from App.js, rather than calling this again